screenpipe-server/src/video.rs

```diff
@@ -169,21 +169,7 @@ async fn start_ffmpeg_process(output_file: &str, fps: f64) -> Result<Child, anyh
         "pad=width=ceil(iw/2)*2:height=ceil(ih/2)*2",
     ];
 
-    if env::consts::OS == "windows" {
-        // TODO switch back to libx264 when ffmpeg is updated in pre_build.js
-        // Use H264_mf encoder for Windows
-        args.extend_from_slice(&[
-            "-vcodec",
-            "h264_mf",
-            "-q:v",
-            "5", // Adjust quality (1-31, lower is better)
-            "-preset",
-            "ultrafast",
-        ]);
-    } else {
-        // Use libx264 for other platforms
-        args.extend_from_slice(&["-vcodec", "libx264", "-preset", "ultrafast", "-crf", "23"]);
-    }
+    args.extend_from_slice(&["-vcodec", "libx265", "-preset", "ultrafast", "-crf", "23"]);
```

Comment:
can you confirm we can still render video on windows in the app chat?

i remember chatgpt telling me using h265 risks some device not being able to render video

Reply:
e.g. just press button search and see if videos render

Reply:
@louis030195 I tested and it works... H265 support is pretty widespread now. AFAIK every device with Windows 10 should be able to run it.